custom-routes-vis/src/main/webapp/script.js

```diff
@@ -12,9 +12,105 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/** Creates the world map. */
+// Array holding origin and destination markers.
+let markers = []
+
+/** Creates the world map and markers. */
 function createWorldMap() {
```

Comment:
We want to be as specific/descriptive as possible.

```suggestion
function createWorldMapWithMarkers() {
```